app/gui/src/controller/searcher/component/group.rs

```diff
@@ -182,6 +242,12 @@ impl Group {
         }
     }
 
+    /// The fully qualified name of the components group, or [`None`] if not available.
+    pub fn qualified_name(&self) -> Option<QualifiedName> {
+        let name = self.name.clone();
+        self.project.as_ref().map(|p| QualifiedName { project: p.clone(), name })
+    }
+
```

Comment:
It does not work very well with module groups not being top modules. We pass here only the module name, so nested modules sharing the name will have identical full qualified names.

Reply:
Removed.